Makefile

```diff
@@ -527,14 +527,6 @@ proto-update-deps:
 .PHONY: proto-all proto-checks proto-regen proto-gen proto-format proto-lint proto-check-breaking proto-check-breaking-third-party proto-update-deps proto-update-check
 
 
-##############################
-### Docs
-##############################
-update-swagger-docs: statik proto-swagger-gen
```

Comment:
The `proto-all` and `proto-regen` targets should be updated to have `proto-swagger-gen` instead of `update-swagger-docs`